front/src/routes/scene/edit-scene/actions/LightFadeInParams.jsx

```diff
@@ -0,0 +1,137 @@
+import { Component } from 'preact';
+import { connect } from 'unistore/preact';
+import { Localizer, Text } from 'preact-i18n';
+import Select from 'react-select';
+
+@connect('httpClient', {})
+class LightFadeInParams extends Component {
+  handleChangeDuration = e => {
+    let newValue = Number.isInteger(parseInt(e.target.value, 10)) ? parseInt(e.target.value, 10) : 0;
+    this.setState({ parameters: { ...this.state.parameters, durationValue: newValue}}, () => {
+      this.props.updateActionProperty(this.props.columnIndex, this.props.index, 'parameters', this.state.parameters);
+    })
+  };
+  handleChangeUnit = e => {
+    this.setState({ parameters: { ...this.state.parameters, durationUnit: e.target.value}}, () => {
+      this.props.updateActionProperty(this.props.columnIndex, this.props.index, 'parameters', this.state.parameters);
+    })
+  };
+  handleChangeTargetBrightness = e => {
+    this.setState({ parameters: { ...this.state.parameters, targetBrightness: e.target.value}}, () => {
+      this.props.updateActionProperty(this.props.columnIndex, this.props.index, 'parameters', this.state.parameters);
+    })
+  }
+  getOptions = async () => {
+    try {
+      const devices = await this.props.httpClient.get('/api/v1/device', {
+        device_feature_category: 'light',
+        device_feature_type: 'brightness'
+      });
+      const deviceOptions = devices.map(device => ({
+        value: device.selector,
+        label: device.name
+      }));
+      await this.setState({ deviceOptions });
+      this.refreshSelectedOptions(this.props);
+      return deviceOptions;
+    } catch (e) {
+      console.log(e);
```

Comment:
Accoring to eslint warn, can you `console.error` instead?

Reply:
Done